src/lib/plugins/inventory.js

```diff
@@ -8,6 +8,14 @@ module.exports.player = function (player, serv, { version }) {
   player.heldItem = new Item(256, 1)
   player.inventory = windows.createWindow(1, 'minecraft:inventory', 'inv', 36)
 
+  player.inventory.on('updateSlot', (slot, newItem) => {
+    player._client.write('set_slot', {
+      windowId: 0, // inventory window id
```

Comment:
Its correct, but what about making it more explicit as:
```js
windowId: player.inventory.id
```
You won't need the commentary